src/main/java/bio/terra/service/profile/google/GoogleBillingService.java

```diff
@@ -135,9 +135,9 @@ public boolean assignProjectBilling(BillingProfileModel billingProfile, GooglePr
         ProjectBillingInfo content = ProjectBillingInfo.newBuilder()
             .setBillingAccountName("billingAccounts/" + billingAccountId)
             .build();
-        try {
-            ProjectBillingInfo billingResponse = cloudBillingClient()
-                .updateProjectBillingInfo("projects/" + projectId, content);
+        try (CloudBillingClient cloudBillingClient = cloudBillingClient()) {
```

Comment:
It's nice to see try-finally be used to manage a resource, but it seems easy for someone to forget to do it. Maybe there's a different method name that can indicate that it's creating a resource that must be closed? Since it's a private method scoped inside a class called `GoogleBillingClient` I think it would be OK to drop "cloud" and "billing" as they're a bit redundant. Maybe `openClient()` or `accessClient()`?